data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/util/DirectoryUtils.java

```diff
@@ -0,0 +1,61 @@
+package com.scalar.db.dataloader.cli.util;
+
+import static com.scalar.db.dataloader.cli.constant.ErrorMessage.ERROR_CREATE_DIRECTORY_FAILED;
+import static com.scalar.db.dataloader.cli.constant.ErrorMessage.ERROR_DIRECTORY_WRITE_ACCESS;
+import static com.scalar.db.dataloader.cli.constant.ErrorMessage.ERROR_EMPTY_DIRECTORY;
+
+import com.scalar.db.dataloader.cli.exception.DirectoryValidationException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.commons.lang3.StringUtils;
+
+/** Utility class for validating and handling directories. */
+public class DirectoryUtils {
```

Comment:
```suggestion
public final class DirectoryUtils {
```

A bit better as a utility class?

Reply:
Updated and added in the code.